### ISSUE TITLE
cannot select detections that are in the same location

### ISSUE THREAD

**Issue by swricci** [swricci/labeler#6]
Some boats are "double detected" and there are two detections (one boat, one wake) plotted directly on top of each other. In this case, one of them is a bad detection, but we are unable to select the "wrong" detection to label as bad.

**Comment by alptezbasaran** [comment]
does it matter which?

**Comment by swricci** [comment]
yes. We should be able to delete the detection that is the incorrect class, and keep the detection that has been verified.
